RoofingCompany.sql

```diff
@@ -769,7 +769,7 @@ GO
 
 CREATE VIEW vPartsView
 AS
-SELECT Part.PartName as [Nazwa części], 
+SELECT TOP 100 PERCENT Part.PartName as [Nazwa części], 
```

Comment:
dlaczego?

Reply:
Baz TOP 100 PERCENT nie można wykonać execute w Management Studio, ze względu na ORDER BY które jest kilka linijek niżej

Reply:
jestem przekonany że to z powodu tego dziwnie wyglądającego joina: `FROM Unit INNER JOIN (Maintenance INNER JOIN (Part INNER JOIN MaintPart 
ON Part.IdPart = MaintPart.IdPart) `